csharp/XenSdkSample/GetVariousRecords.cs

```diff
@@ -0,0 +1,129 @@
+﻿/*
+ * Copyright (c) Citrix Systems, Inc.

```

Comment:
Citrix or Cloud ?

Reply:
It's Cloud for new files, but these ones are only transferred across from a different repo, so I think we can leave them as they are.